File: javascript/chap1/michael/MO-1.js
/*
chapter 1
excercise 1
Problem:
Find average of all the elements in a list.
*/

const avg = (numbers) => {
  if (numbers.length > 0) {
    return (
      numbers.reduce((total, item) => total + item, 0) / numbers.length
    ).toFixed(2);
  }
  return undefined;
};

nums = [2, 4, 6, 6, 7, 9, 90, 45, 23, 67, 89];

console.log(`Average: ${avg(nums)}`);

File: python/chap1/michael/MO-1.py
'''
chapter 1
excercise 1
Problem:
Find average of all the elements in a list.

'''


def avg(elements):
  if len(elements) > 0:
    average = sum(elements) / len(elements)
    return round(average, 2)
  return None

if __name__ == '__main__':
  numbers = [1, 3, 4, 5, 6, 6]
  average = avg(numbers)
  print(f'Average: {average}')